odoosync.php

```diff
@@ -177,7 +177,7 @@ function odoosync_civicrm_post($op, $objectName, $objectId, &$objectRef) {
 
   //Organization, Individual
   $isContactOperation = ($objectName == 'Organization' || $objectName == 'Individual')
-    && ($op == 'create' || $op == 'edit' || $op == 'delete'|| $op == 'trash'|| $op == 'restore');
+    && ($op == 'create' || $op == 'edit' || $op == 'update' ||  $op == 'delete'|| $op == 'trash'|| $op == 'restore');
```

Comment:
or just use in_array : 

```php
in_array($op, ['create', 'edit', 'update' ...etc] ) 
```

actually, you may know better but I am not sure if operation check matters here at all, because it seems to me that you are checking all possible operations here so you can just remove the operation check completely (just guessing).

Reply:
I didn't remove this check because in case of not expected operation(which wasn't included in list) the expected data may not be there or incorrect.